README.md

```diff
@@ -1,24 +1,49 @@
+[![Travis CI Build Status](https://travis-ci.org/mozilla/rkv.svg?branch=master)](https://travis-ci.org/mozilla/rkv)
+[![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/lk936u5y5bi6qafb/branch/master?svg=true)](https://ci.appveyor.com/project/mykmelez/rkv/branch/master)
+[![Documentation](https://docs.rs/rkv/badge.svg)](https://docs.rs/rkv/)
+[![Crate](https://img.shields.io/crates/v/rkv.svg)](https://crates.io/crates/rkv)
+
 # rkv
 
-<a href="https://crates.io/crates/rkv">
-    <img src="https://img.shields.io/crates/v/rkv.svg">
-</a>
+The [rkv Rust crate](https://crates.io/crates/rkv) is a simple, humane, typed Rust interface to [LMDB](http://www.lmdb.tech/doc/).
+
+## Use
+
+Comprehensive information about using rkv is available in its [online documentation](https://docs.rs/rkv/), which you can also generate for local consumption:
 
-rkv is a usable Rust wrapper around LMDB.
+```sh
+cargo doc --open
+```
 
-It aims to achieve the following:
+## Build
 
-- Avoid LMDB's sharp edges (e.g., obscure error codes for common situations).
-- Report errors via `failure`.
-- Correctly restrict to one handle per process via a 'manager'.
-- Use Rust's type system to make single-typed key stores (including LMDB's own integer-keyed stores) safe and ergonomic.
-- Encode and decode values via `bincode`/`serde` and type tags, achieving platform-independent storage and input/output flexibility.
+Build this project as you would build other Rust crates:
 
-## Feature choices
+```sh
+cargo build
+```
 
 If you specify the `backtrace` feature, backtraces will be enabled in `failure`
```

Comment:
This is not relevant to this review, but can you talk me through why you're doing this?  @grigoryk just converted Mentat to `failure` and maybe we want to grow this feature as well.

Reply:
When @rnewman added this note to the README (https://github.com/mozilla/rkv/commit/a07fb4e45299e6ae95d15f95d6696d727e14822d), he also added this comment to Cargo.toml:

```toml
# Get rid of failure's dependency on backtrace. Eventually
# backtrace will move into Rust core, but we don't need it here.
```